src/libsyntax/parse/lexer/tokentrees.rs

```diff
@@ -178,8 +178,7 @@ impl<'a> StringReader<'a> {
                 let raw = self.span_src_raw;
                 self.real_token();
                 let is_joint = raw.hi() == self.span_src_raw.lo() && token::is_op(&self.token);
```

Comment:
Oops, I missed that we actually do preserve this information from lexing! Time to update that other comment, I guess.